app/views/steps/hardship/hardship_reason/edit.html.erb

```diff
@@ -6,9 +6,16 @@
 
     <p class="lede"><%=t '.lead_text' %></p>
 
-    <%= step_form @form_object do |f| %>
-      <%= f.text_area :hardship_reason, size: '60x10', class: 'form-control-3-4' %>
-      <%= f.submit class: 'button' %>
-    <% end %>
+    <div id="gfa_main_container" class="uploaded_doc_<%= uploaded_document?(:hardship_reason) %>">
```

Comment:
The "gfa_main_container" is actually identifying a div for a JS positioning hack. Now that we're using it for multiple steps, it needs a more generic name